libs/membership/membership-impl/src/main/kotlin/net/corda/membership/impl/MemberInfoExtension.kt

```diff
@@ -118,5 +130,10 @@ class MemberInfoExtension {
         @JvmStatic
         val MemberInfo.identityKeyHashes: Collection<PublicKeyHash>
             get() = memberProvidedContext.parseSet(IDENTITY_KEY_HASHES)
+
+        /** Denotes whether this [MemberInfo] represents an MGM node. */
+        @JvmStatic
+        val MemberInfo.isMgm: Boolean
+            get() = mgmProvidedContext.parse<String>(IS_MGM).toBoolean()
```

Comment:
```suggestion
            get() = mgmProvidedContext.parseOrNull(IS_MGM) ?: false
```
The property is optional and should default to false. I've added the same in my PR.